### PR TITLE
Documented the Turtles mine

### PR DESCRIPTION
**Pull request by quantum8**

### CODE REVIEW

File: Projects/Foundations/Schemas/Docs-Books/community-data-mines.json [hunk 1]
```diff
@@ -56,14 +56,18 @@
         {
             "style": "Chapter",
             "text": "X->Foundations->Topic->Sparta Data Mine->"
+        },
+		{
+            "style": "Chapter",
```
Comment (gitforkgit) [inline Projects/Foundations/Schemas/Docs-Books/community-data-mines.json:61]: @quantum8 If you have a moment, could you move the opening bracket  to the left so it lines up with the closing brakcet above? It's not important for the functionality but the structure.

File: Projects/Foundations/Schemas/Docs-Topics/T/Turtles/Turtles-Data-Mine/turtles-data-mine-001-turtles.json [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+            "style": "Title",
+            "text": "Turtles on the Charts"
+        },
+		{
+            "style": "Text",
+            "text": "This trading system is a trend following system that attempts to catch a breakout. It is similar to a Donchian Breakout strategy."
+        },
+		{
+            "style": "Link",
+            "text": "Turtles System 1(investopedia)->investopedia.com/articles/trading/08/turtle-trading.asp"
```
Comment (gitforkgit) [inline Projects/Foundations/Schemas/Docs-Topics/T/Turtles/Turtles-Data-Mine/turtles-data-mine-001-turtles.json:19]: Here the same.

**Review by gitforkgit — APPROVED** [review]
@quantum8  @harrellbm  Everything looks quite good. There are only two small things that would look better. If you don`t want to or don`t have the time to change it, just let me know. After it getting merged, i can make the small changes as well. :+1:

File: Projects/Foundations/Schemas/Docs-Topics/T/Turtles/Turtles-Data-Mine/turtles-data-mine-001-turtles.json (hunk 2 above)
Comment (quantum8) [reply Projects/Foundations/Schemas/Docs-Topics/T/Turtles/Turtles-Data-Mine/turtles-data-mine-001-turtles.json:19]: Just had a look and this is interesting @gitforkgit . I've been using templates from existing docs from way back and haven't noticed what's going on until now; I use notepad++ and it looks like my default setup is 4 spaces = 1 tab, whereas github here must parse it differently. The template I use has 2 tabs at opening bracket and 8 spaces for the closing one. 
What shall our standard here be?

Comment (gitforkgit) [reply Projects/Foundations/Schemas/Docs-Topics/T/Turtles/Turtles-Data-Mine/turtles-data-mine-001-turtles.json:19]: @quantum8 @harrellbm You once commented what the standard is or was at that point. Do you know what the standard is right now?

Comment (harrellbm) [reply Projects/Foundations/Schemas/Docs-Topics/T/Turtles/Turtles-Data-Mine/turtles-data-mine-001-turtles.json:19]: I believe it has been 4 across other places in the system. So let's go with that

**Pull request merged**